sdk/core/core-client-rest/src/helpers/clientCertificatePolicy.browser.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PipelinePolicy } from "@azure/core-rest-pipeline";
+export interface CertificateCredential {}
+
+/**
+ * Gets a pipeline policy that adds the client certificate to the HttpClient agent for authentication.
+ * Only supported on NodeJS
+ */
+export function getClientCertificatePolicy(_cert?: CertificateCredential): PipelinePolicy {
+  throw new Error("Client certificate authentication is only supported in NodeJS");
```

Comment:
is this policy required for confidential ledger/service-fabric to work? Just for general discussion, I sometimes wonder whether we should do no-op for browsers, or don't add the policy when in browser environment for policies that is not supported in browser.

Reply:
I agree that it feels weird to throw here, since if it's required to use the service, what is the point of shipping a browser version to begin with?

Reply:
It is not **required** by the service in both services users can use AAD without client certificate to authenticate, In SF a common scenario is also to do unauthenticated calls within a trusted environment (localhost)